src/components/POLine/Item/ItemForm.js

```diff
@@ -251,27 +230,59 @@ class ItemForm extends Component {
               disabled={isOpenedOrder}
             />
           </Col>
-          <Col xs={6}>
+          <Col
+            xs={6}
+            md={3}
+          >
             <Field
               component={TextField}
               fullWidth
-              label={<FormattedMessage id="ui-orders.itemDetails.publisher" />}
-              name="publisher"
-              onChange={(e, value) => this.onChangeField(value, 'publisher')}
+              label={<FormattedMessage id="ui-orders.itemDetails.edition" />}
+              onChange={(e, value) => this.onChangeField(value, 'edition')}
+              name="edition"
               disabled={isOpenedOrder}
             />
           </Col>
-          <Col xs={6}>
+        </Row>
+        <Row>
+          <Col
+            xs={6}
+            md={3}
+          >
+            <FieldDatepicker
+              label={<FormattedMessage id="ui-orders.itemDetails.subscriptionFrom" />}
+              name="details.subscriptionFrom"
+            />
+          </Col>
+          <Col
+            xs={6}
+            md={3}
+          >
+            <FieldDatepicker
+              label={<FormattedMessage id="ui-orders.itemDetails.subscriptionTo" />}
+              name="details.subscriptionTo"
+              disabled={isOpenedOrder}
+            />
+          </Col>
+          <Col
+            xs={6}
+            md={3}
+          >
             <Field
+              label={<FormattedMessage id="ui-orders.itemDetails.subscriptionInterval" />}
+              name="details.subscriptionInterval"
               component={TextField}
+              type="number"
               fullWidth
-              label={<FormattedMessage id="ui-orders.itemDetails.edition" />}
-              onChange={(e, value) => this.onChangeField(value, 'edition')}
-              name="edition"
               disabled={isOpenedOrder}
             />
           </Col>
-          <Col xs={12}>
+        </Row>
+        <Row>
+          <Col
+            xs={12}
+            md={6}
```

Comment:
so it will take half of available screen space?

Reply:
Like on mockups. Another way is to change the column width in `ContributorForm` instead of here. @aliaksei-chumakou what do you think?

Reply:
could you please attach link to mock-up? I can't find it in the ticket

Reply:
https://issues.folio.org/browse/UIOR-281

Reply:
mock-up there is pretty *ucked up. There is only one field for contributors, while they have 2 - name and type. Also they omit Qualifier field for product IDs. I'm pretty sure that it's better to leave xs 12 for both forms

Reply:
Ok. Let's leave `xs=12` for `contributors` and `product id and type`.